python/feast_spark/job_service.py

```diff
@@ -112,12 +114,26 @@ def _whitelisted_projects(self) -> Optional[List[str]]:
             return whitelisted_projects.split(",")
         return None
 
+    @property  # type: ignore
+    @memoized(maxsize=1)
+    def _whitelisted_project_feature_table_pairs(self) -> Optional[List[Tuple[str, str]]]:
+        if self.client.config.exists(opt.WHITELISTED_FEATURE_TABLES_PATH):
+            _whitelisted_feature_tables = self.client.config.get(opt.WHITELISTED_FEATURE_TABLES_PATH)
+            with open(str(opt.WHITELISTED_FEATURE_TABLES_PATH), "r") as whitelist:
+                return [(line.split(":")[0], line.split(":")[-1]) for line in whitelist]
```

Comment:
Can we have documentation how the file should be formatted? or use a well known format